docs/settingup.md

```diff
@@ -1821,6 +1821,171 @@ Create and execute a `changesheet` action for each sheet in an app. This can be
 ---
 </details>
 
+<details>
+<summary>smartsearch</summary>
+
+## SmartSearch action
+
+Perform a Smart Search in Sense app to find suggested selections.
+
+* `searchtextsource`: Source for list of strings used for searching.
+    * `searchtextlist` (default)
+    * `searchtextfile`
+* `searchtextlist`: List of of strings used for searching.
+* `searchtextfile`: File path to file with one search string per line.
+* `pastesearchtext`: 
+    * `true`: Simulate pasting search text.
+    * `false`: Simulate typing at normal speed (default).
+* `makeselection`: Select a random search result.
+    * `true`
+    * `false`
+* `selectionthinktime`: Optional think time before seleciton is made.
+  * `type`: Type of think time
+      * `static`: Static think time, defined by `delay`.
+      * `uniform`: Random think time with uniform distribution, defined by `mean` and `dev`.
+  * `delay`: Delay (seconds), used with type `static`.
+  * `mean`: Mean (seconds), used with type `uniform`.
+  * `dev`: Deviation (seconds) from `mean` value, used with type `uniform`.
+
+
+### Examples
+
+#### Search with one search term
+```json
+{
+    "action": "smartsearch",
+    "label": "one term search",
+    "settings": {
+        "searchtextlist": [
+            "term1"
+        ]
+    }
+}
+```
+
+#### Search with two search terms
```

Comment:
Two search terms, but the label below is "one term search". I think the difference between terms separated by space and separated as an array needs to be clearly explained as I suspect it may be confusing.